src/main/java/frc/robot/Robot.java

```diff
@@ -62,6 +63,7 @@ public void autonomousInit() {
     @Override
     public void autonomousPeriodic() {
         loopTime = Timer.getFPGATimestamp();
+        new ResetRamp(robotContainer.getRamp());
```

Comment:
Putting this in autonomousPeriodic does not seem correct. You would be rescheduling this over and over, 50 times a second.

Reply:
Moved it to AutonomousInit